client/tests/unit/models/disposition-test.js

```diff
@@ -14,62 +12,6 @@ module('Unit | Model | disposition', function(hooks) {
     assert.ok(model);
   });
 
-  test('when participant opts out/submits, statuscode becomes "saved"/"submitted"', async function(assert) {
```

Comment:
Hmm unless there's a good reason we should not be deleting tests in order to get the suite to pass. These are our only defense against regressions (read: grumpy emails).